src/Orleans.SyncWork/Orleans.SyncWork.csproj

```diff
@@ -1,10 +1,10 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>netstandard2.0</TargetFramework>
-    <LangVersion>10</LangVersion>
+    <TargetFramework>net7.0</TargetFramework>

```

Comment:
is there a reason that we *have* to bring in .net7 dependencies and can't rely on .net standard?  

Not knowing a *whole* lot about how the packages work I'm a bit out of my element here.

Wonder if we can just do different directives for producing packages for both.

On the other hand, not sure if it's that big of a deal, could just say "not doing bug fixes for old versions, only for .net7 and forward"

Reply:
I was unable to build with the upgrade Orleans v7 dependencies without upgrading to net7.0. I'm not 100% sure that there isn't another way around it, but if you look at the Orleans v7 packages they are not listed as being compatible with net standard -- only net7.0.

https://www.nuget.org/packages/Microsoft.Orleans.Server
![image](https://user-images.githubusercontent.com/1981049/207983958-54b5cdc3-be10-44cb-a815-53821c8e69bd.png)